Performance/Performance119.md

```diff
@@ -32,7 +32,7 @@ Don't want to mess around with mods? try out Fabulously Optimized [![Modrinth Lo
 | [DashLoader](https://modrinth.com/mod/dashloader) [![Modrinth Logo](/images/platform_icons/Modrinth.png)](https://modrinth.com/mod/dashloader)[![CurseForge Logo](/images/platform_icons/CurseForge.png)](https://www.curseforge.com/minecraft/mc-mods/dashloader)[![Github Logo](/images/platform_icons/Github.png)](https://github.com/TeamQuantumFusion/DashLoader) | Unknown | This mod launches minecraft faster by caching all of the content on first launch and then loading that cache on the next one | alphaqu | Client | Incompatible (2) | [LGPL-3.0](/licenses/Licenses.md#lgpl-30)
 | [Dimensional Threading](https://modrinth.com/mod/dimthread) [![Modrinth Logo](/images/platform_icons/Modrinth.png)](https://modrinth.com/mod/dimthread)[![Github Logo](/images/platform_icons/Github.png)](https://github.com/WearBlackAllDay/DimensionalThreading) | Unknown | Minecraft mod which optimizes the processing of multiple Dimensions, by assigning them independent threads | WearBlackAllDay | Server | Incompatible (2) | [MIT](/licenses/Licenses.md#mit)
 | [Dynamic FPS](https://modrinth.com/mod/dynamic-fps) [![Modrinth Logo](/images/platform_icons/Modrinth.png)](https://modrinth.com/mod/dynamic-fps)[![CurseForge Logo](/images/platform_icons/CurseForge.png)](https://www.curseforge.com/minecraft/mc-mods/dynamic-fps)[![Github Logo](/images/platform_icons/Github.png)](https://github.com/juliand665/Dynamic-FPS) | Unknown | Automatically reduces rendering speed when minecraft is not focused (to 1 FPS) or hidden (no renders at all). | juliand665 | Client | none | [MIT](/licenses/Licenses.md#mit)
-| [Dynamic View](https://www.curseforge.com/minecraft/mc-mods/dynamic-view) [![CurseForge Logo](/images/platform_icons/CurseForge.png)](https://www.curseforge.com/minecraft/mc-mods/dynamic-view)[![Github Logo](/images/platform_icons/Github.png)](https://github.com/ldtteam/DynView) | APTweaks | This is a small/light serverside utility mod to help balancing lag (TPS) and chunk view/load distance. | someaddon | Server | none | [LGPL-3.0](/licenses/Licenses.md#lgpl-30)
+| [Dynamic View](https://beta.curseforge.com/minecraft/mc-mods/dynamic-view-fabric/files) [![CurseForge Logo](/images/platform_icons/CurseForge.png)](https://beta.curseforge.com/minecraft/mc-mods/dynamic-view-fabric/files)[![Github Logo](/images/platform_icons/Github.png)](https://github.com/ldtteam/DynView) | APTweaks | This is a small/light serverside utility mod to help balancing lag (TPS) and chunk view/load distance. | someaddon | Server | none | [LGPL-3.0](/licenses/Licenses.md#lgpl-30)
```

Comment:
Why was this changed from www to beta?

Reply:
Oh ok, sorry for that, for some reason the curseforge links we're redirecting me to the beta ones